source/api/auth/0.9/index.md

```diff
@@ -44,13 +44,14 @@ This is a work in progress. We are actively seeking implementations and feedback
 
 ## 1. Introduction
 
-The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but policy, regulations, or business models can require users to authenticate and be authorized to interact with access-controlled resources. The authentication process could range from a simple click-through agreement to a multi-factor scheme with a secure identity provider.
+The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but internal policies, legal regulations, business models, and other constraints can require users to authenticate and be authorized to interact with some resources. The authentication process could range from a simple restriction by IP address or a click-through agreement, to a multi-factor scheme with a secure identity provider.
 
-Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version.
+Content providers that need to restrict access to their resources may offer alternative versions that go beyond a simple all-or-nothing proposition. These alternative versions could be degraded based on resolution, watermarking, or compression, for example, but are often better than no access at all.
```

Comment:
I think the use of tiers in the intro sets us up for this section:  http://iiif.io/api/auth/0.9/#tiered-access
I prefer the original for that reason.